urllib3/_backends/_loader.py

```diff
@@ -0,0 +1,67 @@
+import sys
+
+from ..backends import Backend as UserSpecifiedBackend
+
+
+def check_for_python_3_5():
```

Comment:
Is this needed?

Reply:
I don't think so. This should be taken care of by the higher level backend checking logic, plus the fact that you can't get into the async code at all unless you're on a new enough python.